app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
```

Comment:
```suggestion
  def full_title page_title
```
ko cần dùng default value cho page_title đâu cu